src/api/v4/error-handling/carto-error.js

```diff
@@ -71,6 +65,14 @@ function transformWindshaftError (error, layers, analysis) {
   return cartoError;
 }
 
+function _transformAjaxError (error) {
```

Comment:
Another personal taste complain. These functions are not "private", they're auxiliary functions that are used by the function that it's exported, but they're not shown outside this module. So there's no need of marking them as protected.

Reply:
agreed, the js scopes give us real privacity, but as long as the file grows the underscore helps to know which methods are "private".

Besides personal taste, the most of the library uses underscores, if we are removing them lets do it everywhere.